packages/url_launcher/url_launcher_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.9
+
+- Fixed invalid routes when opening a `Link` in a new tab
```

Comment:
This needs to be updated to follow [our CHANGELOG style](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style)

Reply:
Here you are.